Sources/MapboxMaps/Foundation/Camera/CameraAnimationsManager.swift

```diff
@@ -189,11 +131,12 @@ public class CameraAnimationsManager {
     @discardableResult
     public func ease(to camera: CameraOptions,
                      duration: TimeInterval,
+                     curve: UIView.AnimationCurve = .easeOut,
```

Comment:
`easeOut` or `easeInOut`? (I agree with `easeOut`)

Reply:
I think it should be `easeOut` too.